segment_data/traffic_data.py

```diff
@@ -2,25 +2,41 @@
 import requests
 import datetime
 import pytz
+import json
 
 
 STRAVA_ACCESS_TOKEN = os.environ['STRAVA_ACCESS_TOKEN']
 TRAFFIC_URL = "https://www.strava.com/api/v3/segments/%s/all_efforts"
 CALIFORNIA_TIMEZONE = "US/Pacific"
+REDWOOD_SEGMENT_INPUT_FILE = 'redwood_segment_input.json'
 
 
-def get_traffic_count(segment_id, time_parameter):
+def build_segment_id_dict(segment_input_file):
+    with open(segment_input_file, 'r') as infile:
+        segments = json.load(infile).get('redwood_segments')
+    segment_id_dict = {}
+    for segment in segments:
+        trailways_id = segment['id']
+        strava_ids = segment['strava_ids']
+        segment_id_dict[trailways_id] = strava_ids
+    return segment_id_dict
+
+
+SEGMENT_ID_DICT = build_segment_id_dict(REDWOOD_SEGMENT_INPUT_FILE)
```

Comment:
Could be neat to use a `frozendict` for this, so you can't accidentally change the values: https://pypi.python.org/pypi/frozendict